install.sh

```diff
@@ -275,30 +286,36 @@ if [[ "$choice" =~ ^(yes|y|Y) ]]; then
     read -p "Enter the database password: " databasePassword
     read -p "Enter the database name: " databaseName
 
-    userExists=$(sudo -u postgres psql -c "SELECT * FROM pg_user WHERE username = '${databaseUsername}'" | grep -c "1 row")
+    userExists=$(sudo -i -u postgres psql -c "SELECT * FROM pg_user WHERE usename = '${databaseUsername}'" | grep -c "1 row")
+    databaseExists=$(sudo -i -u postgres psql -l | grep "${databaseName}" | wc -l)
```

Comment:
This `grep` will match any database that contains the name, e.g. if `databaseName=foo` it will match databases named `foobar` or `testfoo`. Also, because the owner of the database is also displayed it will match if the owner contains `foo`.